db/index.js

```diff
@@ -1,4 +1,5 @@
 var mongoose = require ('mongoose');
+mongoose.Promise = Promise
 var faker = require ('faker');
 
```

Comment:
I'm wondering if these server routes should be in server/index.js instead of db/index.js? I see you export them from db/index.js and import them to server/index.js; I'm just wondering about best practices for writing `(req, res) =>` server routes, and whether they should be kept inside the node server files. Something to look into. Otherwise this all looks great!